chocs_middleware/trace/middleware.py

```diff
@@ -102,18 +102,21 @@ def handle(self, request: HttpRequest, next: MiddlewareHandler) -> HttpResponse:
             "method": str(request.method),
             "path": str(request.path),
             "route": str(request.route.route),
-            "id": request_id,
-            "correlation_id": correlation_id,
-            "causation_id": causation_id,
         }
 
         Logger.set_tag("request", request_tag)
+        Logger.set_tag("x-request-id", request_id)
```

Comment:
In the other logs I think this one is actual just request-id. It also seems to be inside the lambda element, in lambda logs!